lxd/firewall/drivers/drivers_util.go

```diff
@@ -105,3 +110,27 @@ func getOptimisedDNATRanges(forward *AddressForward) map[[2]uint64][2]uint64 {
 
 	return snatRules
 }
+
+// SubnetMask returns the subnet mask of the given network as a string. Both IPv4 and IPv6 are handled.
+func SubnetMask(ipNet *net.IPNet) string {
```

Comment:
From what I can see, this isn't used outside of the `drivers` package so doesn't need to be exported. 
We tend to keep the exported API of a package as small as possible to help reduce the checks required when changing something in the future (remembering that our packages can be imported into third party projects).